examples/all-sky/all_tests.sh

```diff
@@ -1,4 +1,10 @@
 set -eux
+if [[ $(uname) == *"NT"* ]]; then
+    # Rename files ending with .exe to the same name without the extension
+    for file in *.exe; do
+        mv "$file" "${file%.exe}"
```

Comment:
I think that we should not move the files but set the `EXEEXT` variable to `.exe` when on Windows and honour it in the makefiles and scripts:
```makefile
%$(EXEEXT): %.o
	$(FC) $(FCFLAGS) -o $@ $^ $(LDFLAGS) $(LIBS)
```
```bash
./rrtmgp_allsky${EXEEXT} ...
```
and so on.

Reply:
yeap, I like this more and it was also my first implementation, but then I realised I didn't know if something else in your pipelines was consuming these binaries, so I played it safe. Happy to make that change.